src/components/editor/inputs/InputLang.jsx

```diff
@@ -66,6 +74,25 @@ const InputLang = () => {
       setSelectedTransliterationOptions(newTransliterationOptions)
   }, [value, langOptions, scriptOptions, transliterationOptions])
 
+  useEffect(() => {
+    if (_.isEmpty(textValue)) return
+
+    detectLanguage(textValue)
+      .then((resp) => {
+        if (!_.isEmpty(resp) && resp[0].score >= 0.75) {
```

Comment:
Worth stuffing the `0.75` value in config and referencing it here? Or not really worth making the change at this point?

Reply:
My general philosophy is that if a value is only used once and it won't be changed at runtime, then put it with the code rather than config.

Reply:
@justinlittman Good philosophy!